packages/x-components/tailwind.config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
   // See https://github.com/empathyco/x/pull/655#discussion_r948923711
   corePlugins: [
     'alignItems',
+    'backgroundColor',
```

Comment:
We need to enable it to be able to implement the `x-picture-overlay` variant. 
It will need to add an `x-bg-neutral-50/60` gradient in the parent `div`.